src/display/text_layer.js

```diff
@@ -213,10 +526,15 @@ var renderTextLayer = (function renderTextLayerClosure() {
       var styles = this._textContent.styles;
       var textDivs = this._textDivs;
       var viewport = this._viewport;
+      var bounds = [];
+      var enhanceTextSelection = this._enhanceTextSelection;
       for (var i = 0, len = textItems.length; i < len; i++) {
-        appendText(textDivs, viewport, textItems[i], styles);
+        appendText(textDivs, viewport, textItems[i], styles, bounds,
+                     enhanceTextSelection);
```

Comment:
Nit: please decrease the indentation with two spaces on this line, so that the parameters line up; i.e.
```javascript
        appendText(textDivs, viewport, textItems[i], styles, bounds,
                   enhanceTextSelection);
```